runtime/ggma/examples/generate_text/tinyllama/tinyllama.pipeline

```diff
@@ -0,0 +1,10 @@
+decode: |
+  fuse.attention.py < decode_.circle
+      | reshape.io.py input --by_shape [1,16,30,4] [1,16,32,4]
+      | transpose.io.kvcache.py > decode.circle
+
+merge: |
+  merge.circles.py prefill.circle decode.circle
```

Comment:
It will merge two circles into one circle.
In this phase, the weight sharing is handled by pointing the same buffer index for same content of weights.